### ISSUE TITLE
Building from source fails on Linux because of SQLite pointer error.

### ISSUE THREAD

**Issue by lukehayes** [HaxeFoundation/hashlink#711]
I am trying to build the Hashlink binaries from source and I am getting this error:

```
/home/luke/Desktop/HL/hashlink-1.14/libs/sqlite/sqlite.c: In function ‘sqlite_request’:
/home/luke/Desktop/HL/hashlink-1.14/libs/sqlite/sqlite.c:120:58: error: passing argument 5 of ‘sqlite3_prepare16_v2’ from incompatible pointer type [-Wincompatible-pointer-types]
  120 |         if( sqlite3_prepare16_v2(db->db, sql, -1, &r->r, &tl) != SQLITE_OK ) {
      |                                                          ^~~
      |                                                          |
      |                                                          const char **

```

I also get this after but its only a warning:

```
/home/luke/Desktop/HL/hashlink-1.14/libs/sqlite/sqlite.c: In function ‘sqlite_request’:
/home/luke/Desktop/HL/hashlink-1.14/libs/sqlite/sqlite.c:120:58: error: passing argument 5 of ‘sqlite3_prepare16_v2’ from incompatible pointer type [-Wincompatible-pointer-types]
  120 |         if( sqlite3_prepare16_v2(db->db, sql, -1, &r->r, &tl) != SQLITE_OK ) {
      |                                                          ^~~
      |                                                          |
      |                                                          const char **

```

Using `pacman -Qs sqlite` shows:

`local/sqlite 3.46.1-1
    A C library that implements an SQL database engine`

What am I missing here?

**Comment by lukehayes** [comment]
I just tried the nightly build and everything went fine.

I wanted this so that I could compile my Heaps/Hashlink code from C into a binary and that all worked successfully. Hopefully this might help someone in a similar situation in the future.

**Comment by tobil4sk** [comment]
This was fixed in #686.
